template/base/tsconfig.json

```diff
@@ -5,7 +5,8 @@
     "paths": {
       "src": ["src"]
     },
-    "target": "es2020"
+    "target": "es2020",
+    "resolveJsonModule": true
```

Comment:
Is it okay if we leave this here while I upstream it to seek-oss/tsconfig-seek?
The template infra tests will not work until it is sorted out.
I've tried upstreaming to config/tsconfig.json but the resulting template doesn't seem to take up the config.

Reply:
I'd like to try this in `config/tsconfig.json`. It might be a bit funky locally because you may be referencing the release version of skuba which doesn't include your unreleased changes, but it should work once we release v3.13.0.

Reply:
That explains it. Thanks!